packages/fxa-settings/src/pages/Signin/mocks.tsx

```diff
@@ -185,13 +184,20 @@ export function mockGqlAvatarUseQuery() {
 }
 
 export function mockGqlBeginSigninMutation(
-  opts: { keys: boolean; originalLoginEmail?: string; service?: 'sync' },
+  opts: {
+    keys: boolean;
+    originalLoginEmail?: string;
+    service?: string;
+    unblockCode?: string;
+  },
   inputOverrides: any = {}
 ) {
   const result = opts.keys
     ? createBeginSigninResponse({
         keyFetchToken: MOCK_KEY_FETCH_TOKEN,
-        unwrapBKey: MOCK_UNWRAP_BKEY,
+        // This doesn't actually come back from the server. We have to 'patch it' with
+        // the current client side credentials after the request comes back.
+        // unwrapBKey: MOCK_UNWRAP_BKEY,
```

Comment:
Should this line be commented out?

Reply:
Yeah, it seems like it's a bad mock to me. That would never actually be returned form the server. Without commenting this out, we could get a false positive.